BuyMeCoffee/db/migrate/20150620174417_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration
+  def change
+    create_table :categories do |t|
+      t.string :topic, null: false
```

Comment:
You may want an index here, if you're planning on searching or autocompleting categories at some point.